tools/bumper/bumper.go

```diff
@@ -118,7 +130,7 @@ func initLog() *log.Logger {
 }
 
 func initFlags(paramArgs *inputParams) {
-	flag.StringVar(&paramArgs.componentsConfigPath, "config-path", "", "Full path to components yaml")
+	flag.StringVar(&paramArgs.componentsConfigPath, "config-path", "", "relative path to components yaml from bumping repo")
```

Comment:
ditto, let's remove this argument.

Reply:
ditto [comment](https://github.com/kubevirt/cluster-network-addons-operator/pull/616#discussion_r504485958)